### PR TITLE
[#226] Fix IntegrityError on PublicPropertyForm

### PR DESCRIPTION
**Pull request by farhanmasud**
Fixes [#226](https://github.com/LesFruitsDefendus/saskatoon-ng/issues/226)

Added `min_value` for the fields that were causing the error -

`avg_nb_required_pickers`
`number_of_trees`
`fruits_height`

Added `min_value` for the fields that logically shouldn't allow negative numbers - 

`street_number`
`complement`

### CODE REVIEW

File: saskatoon/harvest/forms.py [hunk 1]
```diff
@@ -470,6 +470,14 @@ class Meta:
         required=False
     )
 
+    def __init__(self, *args, **kwargs):
+        super(PublicPropertyForm, self).__init__(*args, **kwargs)
```
Comment (tristanlatr) [inline saskatoon/harvest/forms.py:474]:
```suggestion
        super().__init__(*args, **kwargs)
```

**Review by tristanlatr — APPROVED** [review]
Thanks a lot, only a minor suggestion.

File: saskatoon/harvest/forms.py (hunk 1 above)
Comment (farhanmasud) [reply saskatoon/harvest/forms.py]: @tristanlatr thanks a lot for your suggestion, I have updated it. Didn't know what I used was originally Python 2 convention and we can use `super` without arguments in Python 3! Thanks!

**Comment by tristanlatr** [comment]
Hi @farhanmasud, 

Can you resolve the conflicts on this PR :)

Thanks

**Comment by farhanmasud** [comment]
@tristanlatr I have resolved the conflicts. Thanks.

**Pull request merged**